extensions/wikia/ARecoveryEngine/js/adBlockRecovery.old.js

```diff
@@ -0,0 +1,100 @@
+/*global define*/
+/**
+ * This is a fallback file kept until cache invalidates.
+ * To be removed in the scope of https://wikia-inc.atlassian.net/browse/ADEN-5065
+ */
+define('ext.wikia.aRecoveryEngine.recovery.sourcePoint', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.aRecoveryEngine.adBlockDetection',
+	'wikia.document',
+	'wikia.instantGlobals',
+	'wikia.log',
+	'wikia.window'
+], function (
+	adContext,
+	adBlockDetection,
+	doc,
+	instantGlobals,
+	log,
+	win
+) {
+	'use strict';
+
+	var customLogEndpoint = '/wikia.php?controller=ARecoveryEngineApi&method=getLogInfo&kind=',
+		logGroup = 'ext.wikia.aRecoveryEngine.adBlockRecovery';
+
+	function getName() {
+		return 'adBlockRecovery';
+	}
+
+	/**
+	 * Mark module as responded not matter if adblock is blocking
+	 * @param callback
+	 */
+	function addResponseListener(callback) {
+		adBlockDetection.addOnBlockingCallback(callback);
+		adBlockDetection.addOnNotBlockingCallback(callback);
+	}
+
+	function getSafeUri(url) {
+		if (isBlocking()) {
+			url = win._sp_.getSafeUri(url);
+		}
+
+		return url;
+	}
+
+	function isEnabled() {
+		var context = adContext.getContext(),
+			enabled = adBlockDetection.isEnabled() &&
+				(context.opts.sourcePointRecovery || context.opts.pageFairRecovery);
+
+		log(['isEnabled', enabled], log.levels.debug, logGroup);
+		return enabled;
+	}
+
+	function track(type) {
+		if (win._sp_ && !win._sp_.trackingSent) {
+			if (Wikia && Wikia.Tracker) {
+				Wikia.Tracker.track({
+					eventName: 'ads.recovery',
+					category: 'ads-recovery-blocked',
+					action: Wikia.Tracker.ACTIONS.IMPRESSION,
+					label: type,
+					trackingMethod: 'analytics'
+				});
+			}
+			if (instantGlobals.wgARecoveryEngineCustomLog) {
+				try {
+					var xmlHttp = new XMLHttpRequest();
+					xmlHttp.open('GET', customLogEndpoint + type, true);
+					xmlHttp.send();
```

Comment:
Please, 'pack it' minimum in some function (even whole ```try ... catch```) it will be much cleaner